src/h5web/explorer/TreeView.tsx

```diff
@@ -39,6 +39,9 @@ function TreeView<T>(props: Props<T>): JSX.Element {
                 onSelect(node);
 
                 if (isBranch) {
+                  if (isExpanded && node !== selectedNode) {
+                    return; // Disable collapsing if the node is not selected
+                  }
```

Comment:
Tu peux tout mettre dans la même condition, non ?

Reply:
I can but it is ugly: `isBranch && (!isExpanded || node === selectedNode)`

Reply:
Perhaps better now: `isBranch && (!isExpanded || isSelected)`

Reply:
yes ! 💯